server/src/main/java/io/crate/types/ByteType.java

```diff
@@ -119,6 +119,11 @@ public int fixedSize() {
         return 16; // object overhead + 1 byte + 7 byte padding
     }
 
+    @Override
+    public long valueSize(Byte value) {
+        return 16;
```

Comment:
Can we align `valueSize` with `fixedSize` for `FixedWidthType`-s - not that critical at the current state but would be nice. 

Say, calling `fixedSize` in `valueSize` in all `FixedWidthType` implementations, as an option

Reply:
I put the methods next to each other to make it clear that it's the same and avoid the indirection/call-stack increase. The JIT would probably inline it anways, but felt like it should be clear enough

Reply:
not insisting, looks ok as is.